tox.ini

```diff
@@ -0,0 +1,32 @@
+[tox]
+envlist = coverage-clean, py27, py33, py34, py35, coverage-report
+
+[testenv]
+commands =
+    coverage run setup.py ftest
+setenv =
+  COVERAGE_FILE=.coverage.{envname}
+deps =
+    {toxinidir}/zope.app.publication-4.0.0a1.dev0.tar.gz
+    coverage
+    zope.testrunner
+    ; ZEO pin to get compatibility for Python 3.3
+    ZEO <5.0
```

Comment:
AFAIK ZEO 5 will continue to work on Python 2.7 with trollius and so it should still work on 3.3 with that same extra dependency. Maybe the early alpha up on PyPI now doesn't indicate that.